unstructured/partition/pdf.py

```diff
@@ -332,7 +332,7 @@ def _partition_pdf_or_image_local(
     model_name = (
         model_name
         if model_name
-        else os.environ.get("UNSTRUCTURED_HI_RES_MODEL_NAME", "detectron2_onnx")
+        else os.environ.get("UNSTRUCTURED_HI_RES_MODEL_NAME", "yolox_quantized")
```

Comment:
It was intentional to stick with the old model to isolate differences in ingest tests diff between the library change itself, and then the model change.

Related PR: https://github.com/Unstructured-IO/unstructured/pull/1607

CC @badGarnet 

The examples/layout-analysis/visualization.py script looks like a useful addition!

Reply:
Got it, Thanks for letting me know! I'll hold this PR until [PR #1607](https://github.com/Unstructured-IO/unstructured/pull/1607) is merged.